arch/arm64/boot/dts/freescale/imx8mp-maivin.dts

```diff
@@ -100,6 +103,7 @@
 		compatible = "st,24c02";
 		pagesize = <16>;
 		reg = <0x57>;
+		status = "okay";
```

Comment:
I had miss the fact that imx8mp-verdin.dtsi has a DT note for the carrier board EEPROM. I'm not sure why though, as it seems to me it would be better placed in the carrier board .dts files. The imx8mp-verdin-dahlia.dtsi file enables it with

```
&eeprom_carrier_board {
        status = "okay";
};
```

We could do the same for the time being. What do you think ?

Reply:
I have the same opinion. It's odd that the device tree for the module describes a EEPROM that would be present on the carrier board.
Will do this